packages/backend/src/ee/services/ai/models/anthropic-claude.ts

```diff
@@ -12,5 +12,10 @@ export const getAnthropicModel = (
 
     const model = anthropic(config.modelName);
 
-    return model;
+    return {
+        model,
+        callOptions: {
+            temperature: config.temperature,
+        },
+    };
```

Comment:
was initially thinking these `callOptions` could be passed in `getModel`'s switch case returns, but this makes it more compartmentalised